Problem_3.py

```diff
@@ -0,0 +1,48 @@
+# Runs on Leetcode
+# BFS
+    # Runtime - O(n)
+    # Memory - O(n)
+# DFS
+    # Runtime - O(n)
+    # Memory - O(1)
+
```

Comment:
Space complexity here will be O(h), if you include the stack

Reply:
Stack as in recursion stack?

Reply:
yup

Reply:
Should we consider it?

Reply:
you can consider it by saying that you are considering the recursion stack